packages/widget/src/components/common/dropdown/dropdown.ts

```diff
@@ -36,12 +36,32 @@ export class Dropdown extends BaseComponent {
   @property({ type: Array })
   options: DropdownOption[] = [];
 
+  @property({ type: String })
+  preSelectedOption = '';
+
   @state()
   selectedOption: DropdownOption | null = null;
 
   @property({ attribute: false })
   onOptionSelected: (option?: DropdownOption) => void = () => {};
 
+  constructor() {
+    super();
+    this._setPreselectedOption();
```

Comment:
isn't this suppose to be done on connected callback?

Reply:
yeah, I can move it to the connected callback as well 👍